login.js

```diff
@@ -0,0 +1,91 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const emailInput = document.querySelector(".emailform .innertext"); // 이메일 input
```

Comment:
💊 제안
innertext라는 클래스명이 어떤 태그인지 잘 알려주지 못하는 것 같아요.
input 태그로 접근하는 것이 좋지 않을까합니다.

```suggestion
  const emailInput = document.querySelector(".emailform input");
```